main.go

```diff
@@ -59,9 +60,6 @@ func main() {
 	opts := options.NewOptions()
 	opts.AddFlags()
 
-	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
this we want to keep, for now we can keep it non cancelable so this would just be

```
ctx := context.Background()
```